crates/tabby/src/serve.rs

```diff
@@ -126,15 +137,16 @@ pub async fn main(config: &Config, args: &ServeArgs) {
 
     #[cfg(feature = "ee")]
     let (api, ui) = if args.webserver {
-        let (api, ui) = tabby_webserver::public::attach_webserver(
-            api,
-            ui,
-            logger,
-            code,
-            args.chat_model.is_some(),
-            args.port,
-        )
-        .await;
+        let (api, ui) = ws
+            .attach_webserver(
+                api,
+                ui,
+                Arc::new(create_logger()),
```

Comment:
why do we still pass in `create_logger` as logger is managed by `ws` now?

Reply:
The webserver still takes `RawEventLogger`, which we can't really implement on the database unless we want to store raw strings or parse out the event data from them. This will still allow completion events to be logged to the database, since they are served through the regular API, and the correct logger will still be sent to the serve process which handles completions.